services/php-fpm-nginx/files/sites-enabled/nginx-site.conf

```diff
@@ -0,0 +1,59 @@
+# This configuration was originally started from what ddev ships with.
+server {
+    listen 80 default_server;
+
+    root /var/lib/tugboat/web;
+
+    index index.php index.htm index.html;
+
+    error_log /dev/stdout info;
+    access_log /dev/stdout;
+
+    location / {
+        absolute_redirect off;
+        try_files $uri $uri/ /index.php?$query_string;
+    }
+
+    location @rewrite {
+        # For D7 and above:
+        # Clean URLs are handled in drupal_environment_initialize().
+        rewrite ^ /index.php;
+    }
+
+    # pass the PHP scripts to FastCGI server listening on socket
+    location ~ \.php$ {
+        try_files $uri =404;
+        fastcgi_split_path_info ^(.+\.php)(/.+)$;
+
+        # Normally we'd use a unix socket here, but the base image is already
+        # configured to listen on a TCP port. Since it's local anyways, we don't
+        # expect any real performance impact.
+        fastcgi_pass localhost:9000;
+
+        fastcgi_buffers 16 16k;
+        fastcgi_buffer_size 32k;
+        fastcgi_param SCRIPT_FILENAME $document_root$fastcgi_script_name;
+        fastcgi_param SCRIPT_NAME $fastcgi_script_name;
+        fastcgi_index index.php;
+        include fastcgi_params;
+        fastcgi_intercept_errors off;
+        # fastcgi_read_timeout should match max_execution_time in php.ini
+        fastcgi_read_timeout 10m;
+        fastcgi_param SERVER_NAME $host;
+        fastcgi_param HTTPS $fcgi_https;
+    }
+
+    # Prevent clients from accessing hidden files (starting with a dot)
+    # This is particularly important if you store .htpasswd files in the site hierarchy
+    # Access to `/.well-known/` is allowed.
+    # https://www.mnot.net/blog/2010/04/07/well-known
+    # https://tools.ietf.org/html/rfc5785
+    location ~* /\.(?!well-known\/) {
+        deny all;
+    }
+
+    # Prevent clients from accessing to backup/config/source files
+    location ~* (?:\.(?:bak|conf|dist|fla|in[ci]|log|psd|sh|sql|sw[op])|~)$ {
+        deny all;
+    }
+}
```

Comment:
It's been 8 years since I've touched a Drupal nginx config. We might want someone else to review this. Also, can you link to the ddev source you are using in the comment at the top?

Reply:
looks good to me :+1: